internal/datanode/buffer.go

```diff
@@ -203,12 +200,26 @@ type Item struct {
 	// The index is needed by update and is maintained by the heap.Interface methods.
 }
 
+// String format Item as <segmentID=0, memorySize=1>
+func (i *Item) String() string {
+	return fmt.Sprintf("<segmentID=%d, memorySize=%d>", i.segmentID, i.memorySize)
+}
+
 // A PriorityQueue implements heap.Interface and holds Items.
 // We use PriorityQueue to manage memory consumed by del buf
 type PriorityQueue struct {
 	items []*Item
 }
 
+// String format PriorityQueue as [item, item]
+func (pq *PriorityQueue) String() string {
+	var items []string
+	for _, item := range pq.items {
+		items = append(items, item.String())
+	}
+	return fmt.Sprintf("[%s]", strings.Join(items, ","))
```

Comment:
Will this list be very long?

Reply:
These two string are for debug only, won't be logging out